mozilla_version/test/test_gecko.py

```diff
@@ -330,6 +330,36 @@ def test_fennec_version_supports_released_edge_cases(version_string):
             Class.parse(version_string)
 
 
+@pytest.mark.parametrize('version_string, raises', (
+    ('68.0a1', False),
+    ('68.0b3', False),
+    ('68.0b17', False),
+    ('68.0', False),
+    ('68.0.1', False),
+    ('68.1b2', False),
+    ('68.1.0', False),
+    ('68.1b3', False),
+    ('68.1.1', False),
+    ('68.2b1', False),
+
+    ('68.0.1b1', True),
+    ('68.1.1b2', True),
+
+    ('69.0a1', True),
+    ('69.0b3', True),
+    ('69.0', True),
+    ('69.0.1', True),
+
+    ('70.0', True),
+))
+def test_fennec_version_ends_at_68(version_string, raises):
```

Comment:
I'm a fan of using a method like https://docs.pytest.org/en/latest/example/parametrize.html#parametrizing-conditional-raising for this, rather than `if .. else` to call the same function... but not-blocking

Reply:
Oooooh TIL! I've used this way because that's what we do on scriptworker and most *script. I'll change this. Thanks!